ezbake/config/conf.d/webserver.conf

```diff
@@ -1,5 +1,6 @@
 webserver: {
+    access-log-config = /etc/puppetserver/request-logging.xml
```

Comment:
@jeffmccune With the pathing changes, should this target `/etc/puppetlabs/puppetserver/request-logging.xml`?